ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -215,7 +215,17 @@ - (void)webView:(__unused WKWebView *)webView decidePolicyForNavigationAction:(W
     }
   }
 
-  if (isJSNavigation) {
+  UIApplication *app = [UIApplication sharedApplication];
+  if ([self shouldOpenInExternalApp:url])
```

Comment:
Can we adhere to the current style of the file? Braces on same line, spacing before/after braces.